src/model/validator.ts

```diff
@@ -12,6 +12,10 @@ export type Validator = {
 	totalDailyReturn: bigint;
 	validatorStake: bigint;
 	validatorReturn: bigint;
+	registrations: string;
+	parsedRegistrations?: bigint[];
+	validatorPermits: string;
+	parsedValidatorPermits?: bigint[];
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this? this variable is used.